src/@next/Switch/Switch.tsx

```diff
@@ -85,6 +87,7 @@ export const Switch = React.forwardRef<HTMLInputElement, SwitchProps>(
         />
         <SwitchStyle
           data-disabled={disabled}
```

Comment:
should we disable this as well?

Reply:
No, we should not disable this. When is in loading we disable the switch to be clickable but the UI does not change tp disabled state.